.storybook/polaris-readme-loader.js

```diff
@@ -194,7 +194,7 @@ import {
   UnstyledLink,
   VisuallyHidden,
   VideoThumbnail,
-  useIndexResourceState,
+  useIndexResourceStateUnstable as useIndexResourceState,
```

Comment:
Used suffix for the hook - since hooks always start with `use` as per convention